sources/OpenMcdf.Extensions/OLEProperties/OLEPropertiesContainer.cs

```diff
@@ -248,7 +248,7 @@ public void Save(CFStream cfStream)
                     PropertyContext = UserDefinedProperties.Context
                 };
 
-                ps.FMTID1 = new Guid("{D5CDD502-2E9C-101B-9397-08002B2CF9AE}");
+                ps.FMTID1 = new Guid(WellKnownFMTID.FMTID_UserDefinedProperties);
```

Comment:
Noticed whilst testing that this was writing the DocSummaryInformation guid into the user defined properties section, when it should be the UserDefinedProperties guid.
I also changed it to use the 'WellKnownFMTID' values instead of duplicating the strings, though that's not required for the fix